index.html

```diff
@@ -221,6 +221,87 @@ <h1 class="post">Multi-Post Stories</h1>
           <button class="bt" type="button">see project</button>
         </div>
       </section>
+      <section class="contact">
+        <h1 class="co">CONTACT US</h1>
+        <p class="descrip">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form class="form" action="#" method="post">
+          <ul class="fo">
+            <li class="remo">
+              <input
+                class="te"
+                type="text"
+                id="name"
+                name="user"
+                maxlength="30"
+                placeholder="Text input"
+                required
+              />
+            </li>
+            <li class="remo">
+              <input
+                class="te"
+                type="email"
+                id="mail"
+                name="user_email"
+                placeholder="fabienswift@gmail.com"
+                required
+              />
+            </li>
+            <li class="">
+              <textarea
+                class="ta"
+                id="txt"
+                name="user_txt"
+                maxlength="500"
+                placeholder="Enter text here"
+                required
+              ></textarea>
+            </li>
+          </ul>
+        </form>
+        <button class="bt" type="button">see project</button>
+      </section>
```

Comment:
- [ ] Please follow the design guideline for this submit button  👇 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/165408505-ba18541f-6f69-4888-b4bd-32751bd5dd06.png)

### Figma Design
![image](https://user-images.githubusercontent.com/46839548/165408435-bb9c088e-518d-4303-a8dc-f22b20b2d31e.png)

- [ ] The form validation doesn't work when the button is clicked. This is likely because the submit button is placed outside of the `<form>...</form>`. Please place the submit button inside the form. Also give it a `type="submit"` 👍 
- [ ] The button text should be exactly as provided in the Figma design